### ISSUE TITLE
provider/azurerm: azurerm_network_interface does not update load_balancer_backend_address_pools_ids

### ISSUE THREAD

**Issue by carinadigital** [hashicorp/terraform#9741]
When trying to update a `azurerm_network_interface` to add it to a loadbalancer backend pool after initial creation, no diff is present.

### Terraform Version
Terraform v0.7.7

### Affected Resource(s)
azurerm_network_interface

### Terraform Configuration Files
```hcl
resource "azurerm_resource_group" "test" {
    name = "LoadBalancerRG"
    location = "West US"
}

resource "azurerm_public_ip" "test" {
    name = "PublicIPForLB"
    location = "West US"
    resource_group_name = "${azurerm_resource_group.test.name}"
    public_ip_address_allocation = "static"
}

resource "azurerm_lb" "test" {
    name = "TestLoadBalancer"
    location = "West US"
    resource_group_name = "${azurerm_resource_group.test.name}"

    frontend_ip_configuration {
      name = "PublicIPAddress"
      public_ip_address_id = "${azurerm_public_ip.test.id}"
    }
}

resource "azurerm_lb_backend_address_pool" "test" {
  location = "West US"
  resource_group_name = "${azurerm_resource_group.test.name}"
  loadbalancer_id = "${azurerm_lb.test.id}"
  name = "BackEndAddressPool"
}


resource "azurerm_virtual_network" "test" {
    name = "acctvn"
    address_space = ["10.0.0.0/16"]
    location = "West US"
    resource_group_name = "${azurerm_resource_group.test.name}"
}

resource "azurerm_subnet" "test" {
    name = "acctsub"
    resource_group_name = "${azurerm_resource_group.test.name}"
    virtual_network_name = "${azurerm_virtual_network.test.name}"
    address_prefix = "10.0.2.0/24"
}

resource "azurerm_network_interface" "test" {
    name = "acctni"
    location = "West US"
    resource_group_name = "${azurerm_resource_group.test.name}"

    ip_configuration {
        name = "testconfiguration1"
        subnet_id = "${azurerm_subnet.test.id}"
        private_ip_address_allocation = "dynamic"
        //load_balancer_backend_address_pools_ids = ["${azurerm_lb_backend_address_pool.test.id}"]
    }
}
```

### Expected Behavior
When uncommenting `load_balancer_backend_address_pools_ids` after initial creation, I should see a plan with a change to the `azurerm_network_interface`.

### Actual Behavior
An empty plan

### Steps to Reproduce
1. `terraform apply`
2. uncomment `load_balancer_backend_address_pools_ids`  and `terraform plan`
3. See there is no diff in the plan.
4. Manually delete `azurerm_network_interface` and `terraform apply`
5. Observe that the `azurerm_network_interface`  is in the correct loadbalancer backend pool.

### Important Factoids
1. `load_balancer_backend_address_pools_ids` is computed https://github.com/hashicorp/terraform/blob/master/builtin/providers/azurerm/resource_arm_network_interface_card.go#L101
2. `azurerm_network_interface`'s loadbalancer settings are never set in resourceArmNetworkInterfaceRead()
https://github.com/hashicorp/terraform/blob/master/builtin/providers/azurerm/resource_arm_network_interface_card.go#L264

**Comment by pmcatominey** [comment]
This was fixed in #9207 which landed in 0.7.8, confirmed on `master` just now:

```
~ azurerm_network_interface.test
    ip_configuration.197258848.load_balancer_backend_address_pools_ids.#:          "0" => "1"
    ip_configuration.197258848.load_balancer_backend_address_pools_ids.2622598790: "" => "/subscriptions/00000/resourceGroups/peter-testing-nic/providers/Microsoft.Network/loadBalancers/TestLoadBalancer/backendAddressPools/BackEndAddressPool"
    ip_configuration.197258848.load_balancer_inbound_nat_rules_ids.#:              "" => "<computed>"
    ip_configuration.197258848.name:                                               "" => "testconfiguration1"
    ip_configuration.197258848.private_ip_address:                                 "" => "<computed>"
    ip_configuration.197258848.private_ip_address_allocation:                      "" => "dynamic"
    ip_configuration.197258848.public_ip_address_id:                               "" => "<computed>"
    ip_configuration.197258848.subnet_id:                                          "" => "/subscriptions/00000/resourceGroups/peter-testing-nic/providers/Microsoft.Network/virtualNetworks/acctvn/subnets/acctsub"
    ip_configuration.3270921307.name:                                              "testconfiguration1" => ""
    ip_configuration.3270921307.private_ip_address_allocation:                     "dynamic" => ""
    ip_configuration.3270921307.subnet_id:                                         "/subscriptions/00000/resourceGroups/peter-testing-nic/providers/Microsoft.Network/virtualNetworks/acctvn/subnets/acctsub" => ""
```

**Comment by pmcatominey** [comment]
Closing as this has been fixed for a while now.
